tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -4233,6 +4237,14 @@ int main(int argc, char *argv[]) {
     InitInvok.getLangOptions().AllowModuleWithCompilerErrors = true;
   }
 
+  if (!options::ModuleAliases.empty()) {
+    PrintingDiagnosticConsumer PDC;
+    SourceManager SM;
+    DiagnosticEngine Diags(SM);
+    Diags.addConsumer(PDC);
+    InitInvok.setModuleAliasMap(options::ModuleAliases, Diags);
```

Comment:
Should this bail out early if the validation fails?  Could either pass through the return value or check the diagnostics engine for emitted errors.